src/mon/Elector.h

```diff
@@ -44,7 +45,11 @@ class Elector {
    * The Monitor instance associated with this class.
    */
   Monitor *mon;
-
+  /**
+   * The instance of the peer monitor map
+   *
+   */ 
+  MonMap peermap;
```

Comment:
@tchaikov  consider the  branch-predictor  idea.do not need to  new and free every time.